src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs

```diff
@@ -65,6 +63,11 @@ public void BeginRequest(HttpContext httpContext, HostingApplication.Context con
 
             if (context.Activity is Activity activity)
             {
+                // Add some ASP.NET Core specific properties as tags
+                activity.AddTag("aspnet.traceidentifier", httpContext.TraceIdentifier);
```

Comment:
most of the time, people do sampling with activity, so if the activity is sampled, then they'd lose this information... do you see that as a concern?

Reply:
I'm not sure. This also affects activity log correlation as well right?

Reply:
I've removed these completely.